# HG changeset patch
# User Rik <rik@octave.org>
# Date 1380153143 25200
#      Wed Sep 25 16:52:23 2013 -0700
# Node ID 5789ad69c85a27029692c5c85d17d4d7da5f3ee8
# Parent  db92cd6117a930505262844a90ae0fdb059e91be
graphics_toolkit.m: Adjust spacing in docstring.

* scripts/plot/graphics_toolkit.m: Adjust spacing in docstring.

diff --git a/scripts/plot/graphics_toolkit.m b/scripts/plot/graphics_toolkit.m
--- a/scripts/plot/graphics_toolkit.m
+++ b/scripts/plot/graphics_toolkit.m
@@ -16,18 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {@var{name} =} graphics_toolkit ()
 ## @deftypefnx {Function File} {@var{name} =} graphics_toolkit (@var{hlist})
 ## @deftypefnx {Function File} {} graphics_toolkit (@var{name})
 ## @deftypefnx {Function File} {} graphics_toolkit (@var{hlist}, @var{name})
-## Query or set the default graphics toolkit which is assigned to new
-## figures.
+## Query or set the default graphics toolkit which is assigned to new figures.
 ## 
 ## With no inputs, return the current default graphics toolkit.  If the input
 ## is a list of figure graphic handles, @var{hlist}, then return the name
 ## of the graphics toolkit in use for each figure.
 ## 
 ## When called with a single input @var{name} set the default graphics toolkit
 ## to @var{name}.  If the toolkit is not already loaded, it is initialized by
 ## calling the function @code{__init_@var{name}__}.  If the first input
